Makefile

```diff
@@ -262,6 +290,46 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### gen_csv: generate the csv for a newer version
+gen_csv: update_devworkspace_crds update_terminal_manifests
```

Comment:
please consider `gen_terminal_csv` or `terminal_gen_csv`

Reply:
Does it depend on devworkspace-operator files?
I wonder if it makes sense to push that make olm-related tasks to https://github.com/redhat-developer/web-terminal-operator
and for sure we can (if possible) duplicate that tasks but they will just invoke make tasks from ./web-terminal-operator folder. WDYT?

Reply:
It needs the devworkspace crd to be copied in but we can do it that way as well